docs/hosts/lattice-protocols/control-interface.md

```diff
@@ -1,50 +1,591 @@
+lattice for use by the host and other applications.
+
+##### Request
+
+Empty body
+
+##### Response
+
+```json
+{
+  "claims": [
+    {
+      "call_alias": "",
+      "caps": "wasmcloud:httpserver,wasmcloud:blobstore",
+      "contract_id": "",
+      "iss": "ACOJJN6WUP4ODD75XEBKKTCCUJJCY5ZKQ56XVKYK4BEJWGVAOOQHZMCW",
+      "name": "blobby",
+      "rev": "0",
+      "sub": "MBY3COMRDLQYTX2AUTNB5D2WYAH5TUKNIMELDSQ5BUFZVV7CBUUIKEDR",
+      "tags": "",
+      "version": "0.3.0"
+    }
+  ]
+}
+```
+
+Please note that the claims data type is not currently concretely defined. It is basically a
+`map<string, string>` that comes straight from the signed JWTs used by wasmCloud. So for actors,
+only some of the fields will be present and providers will also have a different set of fields. We
+hope to make this type more concrete in the future.
+
+#### Host Inventory
+
+`nats req wasmbus.ctl.{lattice_prefix}.get.{host-id}.inv ''`
+
+Gets the inventory of a given host. This includes all actors and providers running on a host as well
+as some basic metadata about the host.
+
+##### Request
+
+Empty body
+
+##### Response
+
+```json
+{
+  "actors": [
+    {
+      "id": "MBY3COMRDLQYTX2AUTNB5D2WYAH5TUKNIMELDSQ5BUFZVV7CBUUIKEDR",
+      "image_ref": "wasmcloud.azurecr.io/blobby:0.3.0",
+      "instances": [
+        {
+          "annotations": {},
+          "image_ref": "wasmcloud.azurecr.io/blobby:0.3.0",
+          "instance_id": "018b451e-a213-febb-c4be-7e6b5080e3b7",
+          "revision": 0,
+          "max_concurrent": 1
+        }
+      ],
+      "name": "blobby"
+    }
+  ],
+  "host_id": "NCKVAECFVP53CEW7ZF44BZKJHSFL655JRQAUW6JALI5AIPAQTF4PTPKF",
+  "issuer": "CDFLNE6XRCTY7LWJT54G6O7K5ZGBDJ3JVGEMGAZOH2TPS4KR4H2JTTAP",
+  "friendly_name": "empty-paper-9801",
+  "labels": {
+    "hostcore.osfamily": "unix",
+    "hostcore.os": "macos",
+    "hostcore.arch": "aarch64"
+  },
+  "providers": [
+    {
+      "annotations": {},
+      "id": "VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M",
+      "image_ref": "wasmcloud.azurecr.io/httpserver:0.19.1",
+      "contract_id": "wasmcloud:httpserver",
+      "link_name": "default",
+      "name": "HTTP Server",
+      "revision": 0
+    }
+  ]
+}
+```
+
+Please note that actor instances are grouped by annotations. So in the example output above, if
+blobby had also been started with the annotations `foo=bar`, then there would be a second instance
+group with the annotations `{ "foo": "bar" }`.
+
+#### Ping Hosts
+
+`nats req wasmbus.ctl.{lattice_prefix}.ping.hosts ''`
+
+Pings all hosts in the lattice and gathers responses. This is a "scatter/gather" type operation,
+meaning that you'll receive multiple json responses from all hosts until your configured timeout. If
+using a nats client (like the NATS CLI), you'll need to set `--replies 0 --timeout <your timeout>`.
+An example response body can be found below.
```

Comment:
Nit: this feels unnecessary, since we've been including example responses on everything